src/controllers/admin.controller.js

```diff
@@ -5,26 +5,30 @@ const universityServiceAPI = require("../services/at-university-api.service");
 
 // AT-UNIVERSITY - Admin - Index
 adminCtrl.renderIndexAdmin = (req, res) => {
-  res.render("admin/index");
+    res.render("admin/index");
 };
 
 // AT-UNIVERSITY - Admin - Render Course List
 adminCtrl.renderCourseList = async (req, res) => {
     try {
         const responseCourses = await universityServiceAPI.getAllCourses();
         console.log("---> adminCtrl.renderCourseList.getAllCourses");
-        //console.log(responseCourses.data);
         const courses = responseCourses.data;
-        res.render("admin/course/index", { courses });
+        res.render("admin/course", { courses });
     } catch (err) {
-        console.error(err.message);
-        res.render("admin/course/index");
+        //if you need to check more details of the error in console replace ".message" by ".response"
+        console.log(err.message);
+        if (err.response && err.response.data) {
+            let errorMsgs = err.response.data.error;
+            req.flash("error_msg", errorMsgs);
+            res.redirect("/admin");

```

Comment:
is that correct?

Reply:
Yes is correct

Reply:
I think no, should be `res.redirect("/admin/course");`

Reply:
ready 😄